### ISSUE TITLE
VOD position save/restore integration with Twitch

### ISSUE THREAD

**Issue by rakslice** [rakslice/orion#2]
Twitch has an internal `resumewatching` API for storing VOD and live stream last positions that is used from the Twitch website video playback, and I've got code for querying it roughed in in the `resumewatching` branch, however there is no known Twitch API OAuth scope that allows access to it, so it's just parked there for now

**Comment by ahjolinna** [comment]
@rakslice : have u seen this ? https://discuss.dev.twitch.tv/t/resume-watching-with-oauth-3-legs-connection-protocol/10196

**Comment by rakslice** [comment]
No, but I had assumed as much.  Perhaps one day they will want to support VODs in some third party native client and they will make this API public.

**Comment by rakslice** [comment]
... or it's gated by client-id, :/

**Comment by ahjolinna** [comment]
btw. why does this feature need to be using the twitch API? why cant it just work like normal video players do with local video files...aka just have a small database where to cache the previous watch time.

oc twitch api solution would be nice but it cant be done for now so why even bother?

**Comment by ahjolinna** [comment]
it seems this works on the "retro" version, at least locally...which is fine...API lvl support would be nice when/if it comes
